gulp/tasks/example.js

```diff
@@ -3,7 +3,7 @@ var serve = require('gulp-serve');
 var connectInject = require('connect-inject');
 
 gulp.task('example', serve({
-  root: ['./example', './packages/skygear/dist'],
+  root: ['./example2', './packages/skygear/dist'],
```

Comment:
Please update the folder to `./exmaple/next/`, the new example is for the next version. And I will rename the PR title.